docs/data/sdk-plugins.md

```diff
@@ -178,10 +178,70 @@ Use an Enrichment Plugin to modify event properties:
               }
             }
     
-            amplitude.init('API_KEY');
             amplitude.add(new FilterEventsPlugin());
+            amplitude.init('API_KEY');
             ```
-  
+
+
+    ???code-example "Send event level groups (click to expand)"
+       This is an example of how to send event level groups in Ampli V2
+       How to send event level groups in SDKs(not in Ampli) is different. Please check the specific SDKs for the usage.
+
+       === "JavaScript"
+            ```js
+            import * as amplitude from '@amplitude/analytics-browser';
+    
+            import {PluginType} from '@amplitude/analytics-types';
+    
+            class EventLevelGroupPlugin {
+              name = 'group-plugin';
+              type = PluginType.ENRICHMENT;
+    
+              async setup(config) {
+                return undefined;
+              }
+    
+              async execute(event) {
+                  event.groups = event.extra['groups'];
+                  return event;
+              }
+    
+              // Allow other events to be processed and sent to destination plugins
+              return event;
+            }
+    
+            ampli.client.add(new EventLevelGroupPlugin());
```

Comment:
```suggestion
            amplitude.add(new EventLevelGroupPlugin());
```

Reply:
@justin-fiedler Why do we use `amplitude.add()` here? Isn't it an example of adding plugins in Ampli?

Reply:
@justin-fiedler  I thin you mean this plugins and middleware should be use amplitude.xxxx anywhere. And since we already mentioned in ampli overview access plugins and middleware should using `ampli.client`, there will have no confusion. 
But here I mentioned the example is for Ampli V2, that make more sense to use `ampli.client` instead of `amplitude.add`? Or we can keep using `amplitude.add` and and remove the ampli word in the description. Wdyt?